var/spack/repos/builtin/packages/py-deeptools/package.py

```diff
@@ -0,0 +1,44 @@
+##############################################################################
+# Copyright (c) 2013-2016, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/llnl/spack
+# Please also see the NOTICE and LICENSE files for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+
+
+class PyDeeptools(PythonPackage):
+    """deepTools addresses the challenge of handling the large amounts of data
+       that are now routinely generated from DNA sequencing centers."""
+
+    homepage = "https://github.com/fidelram/deepTools"
+    url      = "https://github.com/fidelram/deepTools/archive/2.5.2.tar.gz"
+
+    version('2.5.2', '76d6550f9341e9d751508ae8f6e76310')
+
+    depends_on('python@2.7:', type=('build', 'run'))
+    depends_on('py-setuptools', type='build')
+    depends_on('py-numpy@1.8.0:', type=('build', 'run'))
+    depends_on('py-scipy@0.17.0:', type=('build', 'run'))
+    depends_on('py-py2bit@0.1.0:', type=('build', 'run'))
+    depends_on('py-pybigwig@0.2.1:', type=('build', 'run'))
+    depends_on('py-pysam@0.8:', type=('build', 'run'))
+    depends_on('py-matplotlib@1.4.0:', type=('build', 'run'))
```

Comment:
Missing a dependency on `py-numpydoc`. Also, some of the dependencies should be a slightly newer version:
```python
    install_requires=[                                                          
        "numpy >= 1.9.0",                                                       
        "scipy >= 0.17.0",                                                      
        "matplotlib >= 1.4.0",                                                  
        "pysam >= 0.8.2",                                                       
        "numpydoc >=0.5",                                                       
        "pyBigWig >=0.2.1",                                                     
        "py2bit >= 0.2.0"                                                       
    ],
```